exporter/alibabacloudlogserviceexporter/metricsdata_to_logservice.go

```diff
@@ -51,9 +51,13 @@ func (kv *KeyValues) Swap(i, j int) {
 	kv.keyValues[i], kv.keyValues[j] = kv.keyValues[j], kv.keyValues[i]
 }
 func (kv *KeyValues) Less(i, j int) bool { return kv.keyValues[i].Key < kv.keyValues[j].Key }
-func (kv *KeyValues) Sort()              { sort.Sort(kv) }
+
+func (kv *KeyValues) Sort() {
```

Comment:
Can we add unit tests for this file to demonstrate the sanitization?

Reply:
Done